pulsar-functions/proto/src/main/proto/InstanceCommunication.proto

```diff
@@ -65,7 +65,12 @@ message MetricsData {
     map<string, DataDigest> metrics = 1;
 }
 
+message HealthCheckResult {
```

Comment:
I believe that we periodically already do getFunctionStatus? Do we want to overload that function instead of adding a new one?